projects/infer/apptainer.def

```diff
@@ -24,6 +23,7 @@ Stage: build
 # we'll hard pin it to the value we need.
 # TODO: there's got to be a better solution here
 python -m pip install poetry==1.7.1
+pip install poetry-plugin-export
```

Comment:
What's the need for this?

Reply:
`poetry export` won't be installed by default down the line so just future proofing us